File: sqlparser/src/sql/csgen/SqlQueryCacheStatement.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;

namespace CData.Sql {


public class SqlQueryCacheStatement : SqlStatement {
  private string query;

  public SqlQueryCacheStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) : base(dialectProcessor) {
    tokenizer.NextToken();
    query = tokenizer.NextToken().Value;
    ParserCore.ParseComment(this, tokenizer);
  }

  private SqlQueryCacheStatement(Dialect dialectProcessor) : base(dialectProcessor) {
  }


  public override void Accept(ISqlQueryVisitor visitor) {

  }

  public string GetQuery() {
    return this.query;
  }

  public override Object Clone() {
    SqlQueryCacheStatement obj = new SqlQueryCacheStatement(null);
    obj.Copy(this);
    return obj;
  }

  protected override void Copy(SqlStatement obj) {
    base.Copy(obj);
    SqlQueryCacheStatement o = (SqlQueryCacheStatement)obj;
    this.query = o.query;
  }
}
}

File: sqlparser/src/sql/SqlQueryCacheStatement.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;

namespace CData.Sql {
#*/

import core.ParserCore;

public class SqlQueryCacheStatement extends SqlStatement {
  private String query;

  public SqlQueryCacheStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) throws Exception {
    super(dialectProcessor);
    tokenizer.NextToken();
    query = tokenizer.NextToken().Value;
    ParserCore.parseComment(this, tokenizer);
  }

  private SqlQueryCacheStatement(Dialect dialectProcessor) {
    super(dialectProcessor);
  }


  public /*#override#*/ void accept(ISqlQueryVisitor visitor) throws Exception {

  }

  public String getQuery() {
    return this.query;
  }

  @Override
  public Object clone() {
    SqlQueryCacheStatement obj = new SqlQueryCacheStatement(null);
    obj.copy(this);
    return obj;
  }

  @Override
  protected void copy(SqlStatement obj) {
    super.copy(obj);
    SqlQueryCacheStatement o = (SqlQueryCacheStatement)obj;
    this.query = o.query;
  }
}

